templates/.circleci/config.yml.tpl

```diff
@@ -174,9 +174,8 @@ workflows:
           context: *contexts
           filters:
             branches:
-              ignore: /.*/
-            tags:
-              only: /v[0-9]+(\.[0-9]+)*(-.*)*/
+              only:
+                - main
```

Comment:
Some repos still use `master` as the default branch, I suggest using [`{{ .Git.DefaultBranch }}`](https://github.com/getoutreach/stencil/blob/main/internal/codegen/values.go#L59) instead.